### ISSUE TITLE
Data Model and Query Language doubts

### ISSUE THREAD

**Issue by versant2612** [MillenniumDB/MillenniumDB#11]
I have some doubts about MilleniumDB after reading the paper.

1. Does DG/PDG support undirected edges?
2. Does DG/PDG support nested edges?
3. Is it possible to retrieve edge id with DGQL queries?

**Comment by cirojas** [comment]
> Does DG/PDG support undirected edges?

No, at least, for now. If we find relevant use cases where undirected edges are needed, we might add them in the future, however they are not contemplated in the short term. 

> Does DG/PDG support nested edges?

Yes, we do. However, in our current import implementation you can't define an edge that connects other two edges. This will be possible in the near future, as we want to support multiple import formats.

> Is it possible to retrieve edge id with DGQL queries?

Yes, for example the query `SELECT * MATCH (?x)-[?e]->(?y)` will bind the edge_id to the variable `?e`.

**Comment by WolfgangFahl** [comment]
You might want to setup discussion and move this item there so that you can close issues properly.
